surveys/templates/partials/sidebar.html

```diff
@@ -50,19 +54,16 @@ <h3><b>Just Spaces</b></h3>
                     <li>
                         <a href="{% url 'census-areas-list' %}">- Census Areas</a>
                     </li>
+                    <li>
+                        <a href="/admin/users/justspacesuser/">- Users</a>
+                    </li>
                 </ul>
             </li>
             {% endif %}
             <li>
-                <a href="#">
+                <a href="/">
                     <i class="fas fa-question"></i>
-                    FAQ
-                </a>
-            </li>
-            <li>
-                <a href="#">
-                    <i class="fas fa-paper-plane"></i>
-                    Contact
+                    About
```

Comment:
As long as we're getting rid of pages, is there any reason to preserve this `About` pseudo-link?

Reply:
I was thinking we could bring this page to the attention of UCD and ask them for text, and having this be a very simple public-facing site for users without a login. See https://github.com/datamade/just-spaces/issues/66#issuecomment-503251517

Reply:
Sounds fine to me! Added it to our agenda for our next call.